src/pkg/bundle/inspect.go

```diff
@@ -99,6 +109,35 @@ func (b *Bundle) listImages() error {
 	return nil
 }
 
+func (b *Bundle) listVariables() error {
+	message.HorizontalRule()
+	message.Title("Overrides and Variables:", "configurable helm overrides and Zarf variables by package")
+
+	for _, pkg := range b.bundle.Packages {
+		// get package source
+		sha := strings.Split(pkg.Ref, "@sha256:")[1] // using appended SHA from create!
+		source, err := sources.New(*b.cfg, pkg, zarfTypes.ZarfPackageOptions{}, sha, nil)
+		if err != nil {
+			return err
+		}
+
+		tmpDir, err := zarfUtils.MakeTempDir(config.CommonOptions.TempDirectory)
+		if err != nil {
+			return err
+		}
+		pkgPaths := layout.New(tmpDir)
```

Comment:
not sure if I'm missing it, but are the tmpDirs being cleaned up? If not, you can add `defer os.RemoveAll(tmpDir)` here

Reply:
nice catch. gonna add to `getPackageImages()` as well in that case.